examples/trk_color.rs

```diff
@@ -8,11 +8,15 @@ use docopt::Docopt;
 use trk_io::{Header, Point, Reader, Writer};
 
 static USAGE: &'static str = "
-Color a TrackVis (.trk) file
+Color a TrackVis (.trk) file.
+
+This will add 3 scalars (color_x, color_y, color_z) per point, so the output file will weight
```

Comment:
I would remove `, so the output file will weight around the double as the input file.`